### DISCUSSION TITLE
Clean command - remove only orhaned directories

### DISCUSSION THREAD

**Discussion by prullenbak** [spatie/laravel-medialibrary#2485] [Q&A] [answered]
Hi, maybe this is a feature request, but I don't know if it's already possible. It might even be something that I could contribute, if necessary. Anyway, the question:

is it possible to let the media-library:clean artisan command remove only "orphaned directories" and not "deprecated conversions"? We convert some media types (video) seperately from the built in ones, because we need a different format/resolution of video's after they have been uploaded. So in short: 
- a user uploads a video, laravel-medialibrary creates a thumbnail
- we listen to tehe conversioncompleted event and then convert and transcode the video on our own, storing it in the conversions folder. 

Now, when we run the media-library:clean command, it sees all those conversions as deprecated conversion. This is logical, but a bit incorrect. So it would be very nice if we could tell the command to leave those conversions alone, and only remove the "orphaned directories".

Thanks!

**Comment by freekmurze** [comment, marked as answer]
Your PR has been merged. Thanks!
